docs/integrations/web-servers/heroku.md

```diff
@@ -0,0 +1,359 @@
+---
+id: heroku
+title: Heroku
+sidebar_label: Heroku
+description: The Heroku App is used to monitor heroku infrastructure, applications, metrics and error scenarios to ensure observability into heroku and its components.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/web-servers/Heroku.png')} alt="Heroku Icon" height="50" width="50" />
+
+The Sumo Logic App for Heroku is a logs only app that allows you to monitor your Heroku environment. The preconfigured dashboards present information to monitor heroku infrastructure, applications, metrics and error scenarios to ensure observability into heroku and its components.
+
+## Log types
+
+* Heroku’s [Logplex logging service](https://devcenter.heroku.com/articles/logging) makes it easy to collect logs from its applications and forward them to Sumo. We use a [HTTPS log drain](https://devcenter.heroku.com/articles/log-drains#https-drains) to send [Syslog-formatted](https://datatracker.ietf.org/doc/html/rfc5424#section-6) messages to an HTTPS endpoint via a POST request.
+
+* The Heroku Labs [log-runtime-metrics](https://devcenter.heroku.com/articles/log-runtime-metrics) feature adds experimental support for enabling visibility into load and memory usage for running dynos.
+
+### Sample log message
+
+* A Logplex POST body resembles the following:
+```
+83 <40>1 2012-11-30T06:45:29+00:00 host app web.3 - State changed from starting to up
+119 <40>1 2012-11-30T06:45:26+00:00 host app web.3 - Starting process with command `bundle exec rackup config.ru -p 24405
+```
+
+* Runtime metric logs have the following format:
+```
```

Comment:
```suggestion
``` title="Runtime metric logs"
```

Reply:
This won't work. I prefer keeping it the way it is.